portal/src/app/search/CustomListDialog.jsx

```diff
@@ -83,10 +85,11 @@ class CustomListDialog extends React.Component {
 
     setText = (value) => {
```

Comment:
It's to avoid having to bind setText to this.
When using the ES6 class syntax you have to put this in the constructor if you don't use the `=>` syntax:
```
this.setText = this.setText.bind(this);
```